monitoring/poison_message_queue_finder.py

```diff
@@ -35,6 +42,8 @@ def main():
             f'http://{Config.RABBITMQ_HOST}:{Config.RABBITMQ_HTTP_PORT}/api/queues/{v_host}/{queue_name}',
             auth=HTTPBasicAuth(Config.RABBITMQ_USER, Config.RABBITMQ_PASSWORD)).json()
 
+        queue_details.raise_for_status()
```

Comment:
Fails because queue details is a dictionary, the `requests.get` has the `.json` call chained on it.

Reply:
fixed